src/components/@common/NotificationButton/NotificationButton.tsx

```diff
@@ -0,0 +1,13 @@
+import ActiveIcon from '@/assets/images/notification_on_icon.svg';
+import InactiveIcon from '@/assets/images/notification_off_icon.svg';
+import styles from '@/components/@common/NotificationButton/NotificationButton.module.css';
+
+interface Props {
+  active: boolean;
+}
+
+function NotificationButton({ active }: Props) {
+  return <div className={styles.wrapper}>{active ? <InactiveIcon /> : <ActiveIcon />}</div>;
+}
+
```

Comment:
```suggestion
  return (
    <div className={styles.wrapper}>{active ? <InactiveIcon /> : <ActiveIcon />}
    </div>
    );
}
```
이런 식으로 한줄이더라도 `()`를 사용하는건 어떻게 생각하세요?

Reply:
아마 이건 prettier 설정으로 알아요

Reply:
아 자동으로 변환된거군요